src/styles/blocks/page.css

```diff
@@ -0,0 +1,4 @@
+body {
```

Comment:
This file should be dedicated to styles for a specific BEM block. Consider moving these general styles to a more appropriate file, such as a global styles file, if one exists.